src/command.js

```diff
@@ -2,6 +2,9 @@ import path from 'path'
 import { parseOptions } from './options'
 
 let JQ_PATH = path.join(__dirname, '../bin/jq')
```

Comment:
could you separate this into `path.join(__dirname, '..', 'bin', 'jq')`.
That affects the paths on windows